client/js/lounge.js

```diff
@@ -372,6 +408,16 @@ $(function() {
 			.find("#chan-" + data.chan)
 			.find(".messages");
 
+
+			// remove the date-change marker we put at the top, because it may
```

Comment:
Sorry that became a thing when switching the spaces to tabs to not get yelled at by the linter